src-java/flowhs-topology/flowhs-storm-topology/src/main/java/org/openkilda/wfm/topology/flowhs/bolts/RouterBolt.java

```diff
@@ -74,6 +76,8 @@
 public class RouterBolt extends AbstractBolt {
 
     public static final String FLOW_ID_FIELD = "flow-id";
+    public static final String FIELD_ID_COMMAND = "command";
```

Comment:
this field is unused

Reply:
I have a plan to use "command" patter in flowhs too (same as in network). But it become too big change, so I have postponed this idea. This is artefact from that idea. Removed.